wqflask/wqflask/correlation/show_corr_results.py

```diff
@@ -74,6 +74,193 @@ def set_template_vars(start_vars, correlation_data):
     return correlation_data
 
 
+def apply_filters(trait, target_trait, target_dataset, **filters):
+    def __p_val_filter__(p_lower, p_upper):
+        return not (float(trait.get('corr_coefficient', 0.0)) >= p_lower and
+                    float(trait.get('corr_coefficient', 0.0)) <= p_upper)
+
+    def __min_filter__(min_expr):
+        if (target_dataset['type'] in ["ProbeSet", "Publish"] and target_trait['mean']):
+            return (min_expr != None) and (float(target_trait['mean']) < min_expr)
```

Comment:
Similar comment to above:
```suggestion
            return float(target_trait['mean']) > min_expr != None
```

Reply:
@BonfaceKilz  the above example may lead to unintended results
for example

min_expr = None
``` python
return 2.4 < min_expr is not None

results to 
TypeError: '<' not supported between instances of 'float' and 'str'

```
and if you use bracket that means you will be comparing int to boolean